### ISSUE TITLE
FileNotFoundError

### ISSUE THREAD

**Issue by yochananmarqos** [codywd/NetGUI#59]
I installed the AUR package on Manjaro Cinnamon and the GUI will not launch:

```
$ su -c "netgui"
Traceback (most recent call last):
  File "/usr/share/netgui/main.py", line 559, in <module>
    NetGUI()
  File "/usr/share/netgui/main.py", line 114, in __init__
    self.notifications = Notification()
  File "/usr/share/netgui/Library/notifications.py", line 9, in __init__
    user_prefs = json.load(open(preferences_file))
FileNotFoundError: [Errno 2] No such file or directory: '/var/lib/netgui/preferences.json'

```
All that is in `/var/lib/netgui` is `program.pid`.

**Comment by codywd** [comment]
Hey,

Thanks for the bug report. I'll look into it soon. Pretty clear what is happening, but need to look at why.

**Comment by manouchk** [comment]
Same problem here with netgui 0.85-1

**Comment by codywd** [comment]
I believe this might be fixed in the new push to the develop branch pushed as part of c275c22, but I don't have an Arch Linux system handy at the moment. I'm working on buying an old laptop I can throw Arch on so that I can get back to testing/development of this (or alternatively/preferably, hoping my a new charger will get my Sager gaming laptop up again, and I can just install Arch on it).

If you want to try pulling the develop code and see if it works, that'd be great. Otherwise, I'll test this code once I can procure/build Arch on a laptop again. If it works, I'll merge and update the AUR.

Basically, it isn't pre-creating a default prefs file. Simple problem, hopefully this simple solutions fixes it.

**Comment by yochananmarqos** [comment]
@codywd Yes, it's fixed in the develop branch. 

FYI, I edited the shebang in `setup.py` to `#!/usr/bin/python`  and used `python-setuptools`, `python2-setuptools` was not required.

**Comment by codywd** [comment]
Perfect, any other issues related to the FileNotFound or is it good? I updated setup.py in 35ae570 to reflect the change you made and PKGBUILD in 4d96891
